protos/channel_responses.proto

```diff
@@ -14,22 +14,41 @@ message SiteBannerDetails {
   string description = 2;
   string background_url = 3;
   string logo_url = 4;
-  repeated int32 donation_amounts = 5;
+  repeated double donation_amounts = 5;
   SocialLinks social_links = 6;
 }
 
-enum WalletConnectedState {
-  NO_VERIFICATION = 0;
-  UPHOLD_ACCOUNT_NO_KYC = 1;
-  UPHOLD_ACCOUNT_KYC = 2;
+enum UpholdWalletState {
+  UPHOLD_ACCOUNT_NO_KYC = 0;
+  UPHOLD_ACCOUNT_KYC = 1;
+}
+
+message UpholdWallet {
+  string address = 1;
+  UpholdWalletState wallet_state = 2;
+}
+
+enum PaypalWalletState {
+  PAYPAL_ACCOUNT_NO_KYC = 0;
+  PAYPAL_ACCOUNT_KYC = 1;
+}
+
+message PaypalWallet {
+  string address = 1;
```

Comment:
we don't need this one

Reply:
If we remove this we could put the wallet state fields before the address fields for consistency.

Reply:
agree as we will always have state for all wallets